packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -277,14 +263,24 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
         background: 'none',
         backgroundColor: 'transparent',
         border: 'none',
-        height: DROPDOWN_ITEMHEIGHT,
-        lineHeight: DROPDOWN_ITEMHEIGHT,
+        height: DROPDOWN_ITEM_HEIGHT,
+        lineHeight: DROPDOWN_ITEM_HEIGHT,
         cursor: 'default',
         padding: '0px 16px',
         userSelect: 'none',
         textAlign: 'left'
       }
     ],
-    subComponentStyles: { label: { root: { display: 'inline-block' } } }
+    subComponentStyles: {
+      label: { root: { display: 'inline-block' } },
+      panel: {
+        root: [panelClassName],
+        main: {
+          // Force drop shadow even under medium breakpoint
+          boxShadow: '-30px 0px 30px -30px rgba(0,0,0,0.2)'
+        },
+        contentInner: { padding: '0 0 40px' }
```

Comment:
Guessing the change from 20px to 40px was intentional?

Reply:
Nope... was doing some testing and forgot to change it back. Good catch 👍